tests/test_accounts.py

```diff
@@ -23,7 +23,6 @@ def test_new_post(self, flask_test_client):
 
         assert status_code == 201
         assert "account_id" in response_data.keys()
-        assert "applications" in response_data.keys()
```

Comment:
📓 Worth noting that `"account_id" in response_data` would default to searching keys.

Reply:
Lazy copy and pasting..